config/develop/nextflow-aurora-mysql.yaml

```diff
@@ -1,12 +1,14 @@
 template_path: nextflow-aurora-mysql.yaml
 stack_name: nextflow-aurora-mysql
 dependencies:
+  - common/workflows-kms-key.yaml
```

Comment:
doesn't `nextflow-tower-secret.yaml` need to exist before running this template?

Reply:
No, those secrets are not use in the aurora stack. It has its own.